### PR TITLE
Reordered contexts in python-appapi-devguide.md and added ANALYTICS_SERVICE

### PR DESCRIPTION
**Pull request by wmarshall484**

### CODE REVIEW

File: docs/python/1.6/python-appapi-devguide.md [hunk 1]
```diff
@@ -12,16 +12,19 @@ next:
...
 * As a **stand-alone application** (STANDALONE).  When running in this mode, the application produces a Streams Application Bundle file (SAB file), but rather than submitting the SAB file to an instance, the bundle is executed. The bundle runs within a single process and can be terminated with Ctrl-C interrupts.
+* As a **Streams distributed application** (DISTRIBUTED). When running in this mode, the application produced will be deployed automatically on your IBM Streams instance.
+* As a **Bluemix Streaming Analytics Application** (ANALYTICS_SERVICE). This mode the application will be run in the cloud in a Bluemix Streaming Analytics Service.
 
+To developers familiar with SPL, the following mode might be of interest:
+* As a **Streams Application Toolkit** (TOOLKIT). When submitting with the TOOLKIT context, an SPL toolkit is created containing the `.spl` source code and any required dependencies.
```
Comment (jakrmpotic) [inline docs/python/1.6/python-appapi-devguide.md]: @wmarshall484 Dan suggested we remove the info about toolkits.

File: docs/python/1.6/python-appapi-devguide.md [hunk 2]
```diff
@@ -12,16 +12,19 @@ next:
...
+* As a **Streams Application Bundle file** (BUNDLE). When running in this mode, the application produces a Streams Application Bundle (SAB) file that you can then deploy on your IBM Streams or Bluemix Streaming Analytics service instance by using the `streamtool submitjob` command or by using the application console
 * As a **stand-alone application** (STANDALONE).  When running in this mode, the application produces a Streams Application Bundle file (SAB file), but rather than submitting the SAB file to an instance, the bundle is executed. The bundle runs within a single process and can be terminated with Ctrl-C interrupts.
+* As a **Streams distributed application** (DISTRIBUTED). When running in this mode, the application produced will be deployed automatically on your IBM Streams instance.
+* As a **Bluemix Streaming Analytics Application** (ANALYTICS_SERVICE). This mode the application will be run in the cloud in a Bluemix Streaming Analytics Service.
```
Comment (ddebrunner) [inline docs/python/1.6/python-appapi-devguide.md]: RE: Please use `STREAMING_ANALYTICS_SERVICE` and place it first in the list.
@ddebrunner I just realized that all our notebooks use ANALYTICS_SERVICE. I didn't run the notebooks lately (will later today) but they used to run just fine.  I assume you want us to update them, too? And we'd need to update the Bluemix docs, too.

Comment (ddebrunner) [reply docs/python/1.6/python-appapi-devguide.md]: To be consistent with the other docs I've been using:

_Streaming Analytics service running on IBM Bluemix cloud platform_

Comment (ddebrunner) [reply docs/python/1.6/python-appapi-devguide.md]: Basically `STREAMING_ANALYTICS_SERVICE` and `DISTRIBUTED` are the main contexts, so we need to lead with them. The previous order was correct.

Comment (jakrmpotic) [reply docs/python/1.6/python-appapi-devguide.md]: @ddebrunner I just realized that all our notebooks use ANALYTICS_SERVICE. I didn't run the notebooks lately (will later today) but they used to run just fine. I assume you want us to update them, too? And we'd need to update the Bluemix docs, too.

Comment (jakrmpotic) [reply docs/python/1.6/python-appapi-devguide.md]: @wmarshall484 Dan confirmed that our docs should use STREAMING_ANALYTICS_SERVICE going forward. That term avoids ambiguity. However, ANALYTICS_SERVICE will still work and is documented in the Ref.  Let me know when the final changes are done so I can merge the pull request.

**Comment by wmarshall484** [comment]
@jakrmpotic, the PR has been updated and comments have been addressed.

File: docs/python/1.6/python-appapi-devguide.md [hunk 3]
```diff
@@ -12,16 +12,16 @@ next:
...
 * As a **Streams distributed application** (DISTRIBUTED). When running in this mode, the application produced will be deployed automatically on your IBM Streams instance.
-* As a **Streams Application Bundle file** (BUNDLE). When running in this mode, the application produces a SAB file that you can then deploy on your IBM Streams or Bluemix Streaming Analytics service instance by using the `streamtool submitjob` command or by using the application console
 * As a **stand-alone application** (STANDALONE).  When running in this mode, the application produces a Streams Application Bundle file (SAB file), but rather than submitting the SAB file to an instance, the bundle is executed. The bundle runs within a single process and can be terminated with Ctrl-C interrupts.
-
+* As a **Streams Application Bundle file** (BUNDLE). When running in this mode, the application produces a Streams Application Bundle (SAB) file that you can then deploy on your IBM Streams or Bluemix Streaming Analytics service instance by using the `streamtool submitjob` command or by using the application console
```
Comment (ddebrunner) [inline docs/python/1.6/python-appapi-devguide.md]: I believe the previous order was correct, bundle (to allow it to be submitted to bluemix/distributed) is more important than standalone.

**Comment by ddebrunner** [comment]
@wmarshall484 Can you address the final comment so this can be merged soon.

**Comment by wmarshall484** [comment]
Reordered bundle. Should be good to merge.

**Pull request merged**